### PR TITLE
Clean up plutonium core a bit

### PR DESCRIPTION
**Pull request by xxalpha**
Clean up the code a little bit to maybe prevent a bug from happening?

Fixes https://github.com/tgstation/-tg-station/issues/17804 ?

### CODE REVIEW

File: code/game/objects/items/nuke_tools.dm [hunk 1]
```diff
@@ -11,7 +11,8 @@
 	var/cooldown = 0
 
 /obj/item/nuke_core/New()
-	SSobj.processing += src
+	..()
+	SSobj.processing |= src
```
Comment (optimumtact) [inline code/game/objects/items/nuke_tools.dm]: The mark of a truly paranoid man :smile:

Comment (PKPenguin321) [inline code/game/objects/items/nuke_tools.dm]: I don't get it

File: code/game/objects/items/nuke_tools.dm [hunk 2]
```diff
@@ -41,13 +42,16 @@
...
+	if(istype(core))
 		SSobj.processing -= core
 		icon_state = "core_container_sealed"
-		playsound(loc, 'sound/items/Deconstruct.ogg', 100, 1)
-		if(loc == user)
-			user << "<span class='warning'>Container is sealed, the radiation is contained.</span>"
-	return 1
+		playsound(loc, 'sound/items/Deconstruct.ogg', 60, 1)
+		if(ismob(loc))
+			loc << "<span class='warning'>[src] is permanently sealed, [core]'s radiation is contained.</span>"
```
Comment (optimumtact) [inline code/game/objects/items/nuke_tools.dm]: wouldn't this be better as a visible_message?

Comment (xxalpha) [inline code/game/objects/items/nuke_tools.dm]: I'd rather keep it to the traitor.

Comment (optimumtact) [inline code/game/objects/items/nuke_tools.dm]: oh shoot good point

**Pull request merged**